cicd/gitops/configuring-openshift-login-for-argo-cd.adoc

```diff
@@ -0,0 +1,26 @@
+[id="configuring-openshift-login-for-argo-cd"]
+= Configuring OpenShift login for Argo CD
+include::modules/common-attributes.adoc[]
+include::modules/gitops-document-attributes.adoc[]
+:context: configuring-openshift-login-for-argo-cd
+
+toc::[]
+
+Argo CD allows you to log in by using your OpenShift Credentials through Keycloak as an Identity Broker.
```

Comment:
Who is the end user for this assembly? Is it Admin or is it the GitOps developer team. If it is admin, then we need to specify if this is something the cluster admin does.
If this is for the GitOps developer, maybe this should live in their GH repo, and not in end user docs.

Reply:
Suggest adding a sentence that explains what needs to be done to log in by using your OpenShift Credentials through Keycloak as an Identity Broker. For example: To log in by using your OpenShift Credentials through Keycloak, you need to:

1. Create a client and get the secret
2. Configure Argo CD OpenID Connect (OIDC) using the secret... 
This will help tie the pieces together.

Reply:
@Preeticp Keycloak is for user management, so the tagret audience is whoever administrates the cluster (may but must not be a cluster admin).

>More importantly, maybe we should move this assembly before the Configuring a cluster with Red Hat OpenShift GitOps assembly on the right panel?
We can but it doesn't matter. What I think we can add is a clarification;

When Argod CD Operator is installed, there is a ready-to-use instance with `admin` role that one can use with to additional restrictions (this is the Logging in to Argo CD... procedure). If the same cluster is to be used by many users (a team, for examle), an admin might want to set up an SSO and manage the users with the Keycloak. This is what the assembly is about.